yml/apdb.yaml

```diff
@@ -0,0 +1,2704 @@
+    datatype: long
+    nullable: false
+    description: Id of the second-closest nearby object.
+    mysql:datatype: BIGINT
+    ivoa:ucd: meta.id;src
+  - name: nearbyObj2Dist
+    "@id": "#DiaObject.nearbyObj2Dist"
+    datatype: float
+    description: Distance to nearbyObj2.
+    mysql:datatype: FLOAT
+    fits:tunit: arcsec
+  - name: nearbyObj2LnP
+    "@id": "#DiaObject.nearbyObj2LnP"
+    datatype: float
+    description: Natural log of the probability that the observed diaObject is the
+      same as the nearbyObj2.
+    mysql:datatype: FLOAT
+  - name: nearbyObj3
+    "@id": "#DiaObject.nearbyObj3"
+    datatype: long
+    nullable: false
+    description: Id of the third-closest nearby object.
+    mysql:datatype: BIGINT
+    ivoa:ucd: meta.id;src
+  - name: nearbyObj3Dist
+    "@id": "#DiaObject.nearbyObj3Dist"
+    datatype: float
+    description: Distance to nearbyObj3.
+    mysql:datatype: FLOAT
+    fits:tunit: arcsec
+  - name: nearbyObj3LnP
+    "@id": "#DiaObject.nearbyObj3LnP"
+    datatype: float
+    description: Natural log of the probability that the observed diaObject is the
+      same as the nearbyObj3.
+    mysql:datatype: FLOAT
+  # - name: nearbyExtObj1
+  #   "@id": "#DiaObject.nearbyExtObj1"
+  #   datatype: long
+  #   description: Id of the closest extended DR Object by second moment-based separation.
+  #   mysql:datatype: BIGINT
+  #   ivoa:ucd: meta.id;src
+  # - name: nearbyExtObj1Sep
+  #   "@id": "#DiaObject.nearbyExtObj1Sep"
+  #   datatype: float
+  #   description: Second moment-based separation of nearbyExtObj1 (unitless).
+  #   mysql:datatype: FLOAT
+  # - name: nearbyExtObj2
+  #   "@id": "#DiaObject.nearbyExtObj2"
+  #   datatype: long
+  #   description: Id of the second closest extended DR Object by second moment-based separation.
+  #   mysql:datatype: BIGINT
+  #   ivoa:ucd: meta.id;src
+  # - name: nearbyExtObj2Sep
+  #   "@id": "#DiaObject.nearbyExtObj2Sep"
+  #   datatype: float
+  #   description: Second moment-based separation of nearbyExtObj2 (unitless).
+  #   mysql:datatype: FLOAT
+  # - name: nearbyExtObj3
+  #   "@id": "#DiaObject.nearbyExtObj3"
+  #   datatype: long
+  #   description: Id of the third closest extended DR Object by second moment-based separation.
+  #   mysql:datatype: BIGINT
+  #   ivoa:ucd: meta.id;src
+  # - name: nearbyExtObj3Sep
+  #   "@id": "#DiaObject.nearbyExtObj3Sep"
+  #   datatype: float
+  #   description: Second moment-based separation of nearbyExtObj3 (unitless).
+  #   mysql:datatype: FLOAT
+  # - name: nearbyLowzGal
+  #   "@id": "#DiaObject.nearbyLowzGal"
+  #   datatype: string
+  #   length: 64
+  #   description: External catalog name of the nearest low-z potential host.
+  #   mysql:datatype: VARCHAR(64)
+  # - name: nearbyLowzGalSep
+  #   "@id": "#DiaObject.nearbyLowzGalSep"
+  #   datatype: float
+  #   fits:tunit: arcsec
+  #   description: Separation distance of nearbyLowzGal.
+  #   mysql:datatype: FLOAT
```

Comment:
Bunch of columns are commented out, these were added recently to `baselineSchema`. If they are not useful in APDB I can remove them, otherwise I can un-comment them.

Reply:
I have uncommented these new columns.

Another small change since review - `length` attribute added to `timestamp` and `binary` columns. `length` is not used currently by APDB classes, it is mostly to suppress `felis` messages.